src/ocean/sys/CloseOnExec.d

```diff
@@ -53,11 +53,9 @@
     This does not affect `stdin`, `stdout` and `stderr`, which are opened before
     the start of the program.
 
-    Currently the flag is `false` to keep the system's default behaviour. In
-    general it is recommended to set it to `true` to avoid the aforementioned
-    problem unless file descriptor inheriting is needed. Since there appears to
-    be no use case for it for us the default value of `open_with_close_on_exec`
-    will change to `true` in the next major ocean release.
+    Currently the flag is `true`, which is in general recommended to avoid the
```

Comment:
Please remove the word "Currently", I wrote it to imply that this will change in a future release.